lib/purgatory/purgatory.rb

```diff
@@ -47,6 +47,7 @@ def approve!(approver = nil)
     return false if approved? 
     if soul_with_changes.save
       soul.send(performable_method[:method],*performable_method[:args]) if performable_method
+      return false if soul.errors.any?
```

Comment:
the idea is to add a method to reverse loan that adds an error if transfers have been sent out. so the error won't be added until after reverse loan is performed, if we did it in the financeit code approve! would succeed correct?  it would then get moved out of the awaiting review queue even though the method wasn't actually executed I think